Development/nmos/ocsp_utils.cpp

```diff
@@ -16,58 +16,65 @@ namespace nmos
 
         namespace details
         {
-            // construct OCSP request using the issuer certificate and the list of server certificates
+            typedef std::map<std::string, std::vector<std::string>> issuer_certificate_vs_server_certifiactes_map;
```

Comment:
Typo "certifiactes" appears all over the place.

Reply:
Fixed